src/soilbiogeochem/SoilBiogeochemDecompCascadeBGCMod.F90

```diff
@@ -302,6 +381,11 @@ subroutine init_decompcascade_bgc(bounds, soilbiogeochem_state_inst, soilstate_i
             f_s1s3(c,j) = .004_r8 / (1._r8 - t)
             rf_s1s2(c,j) = t
             rf_s1s3(c,j) = t
+            ! TODO check [kaveh] why do we start with 0?
+            docf_s1s2(c,j) = 0.0_r8
+            docf_s1s3(c,j) = 0.0_r8
+            docf_s1s2(c,1) = 0.0007143_r8
```

Comment:
@devarajun where are these numbers coming from? Do they need to be hard-coded?

Reply:
These numbers should be read from netcdf paramafile in the inputdata. For now, I put the numbers in the code.